nilearn/plotting/displays/_projectors.py

```diff
@@ -136,59 +201,10 @@ def add_graph(
         # make the lines below well-behaved
         adjacency_matrix = np.nan_to_num(adjacency_matrix)
 
-        # safety checks
-        if "s" in node_kwargs:
-            raise ValueError(
-                "Please use 'node_size' and not 'node_kwargs' "
-                "to specify node sizes"
-            )
-        if "c" in node_kwargs:
-            raise ValueError(
-                "Please use 'node_color' and not 'node_kwargs' "
-                "to specify node colors"
-            )
-
-        adjacency_matrix_shape = adjacency_matrix.shape
-        if (
-            len(adjacency_matrix_shape) != 2
-            or adjacency_matrix_shape[0] != adjacency_matrix_shape[1]
-        ):
-            raise ValueError(
-                "'adjacency_matrix' is supposed to have shape (n, n)."
-                f" Its shape was {adjacency_matrix_shape}"
-            )
-
-        node_coords_shape = node_coords.shape
-        if len(node_coords_shape) != 2 or node_coords_shape[1] != 3:
-            message = (
-                "Invalid shape for 'node_coords'. "
-                "You passed an 'adjacency_matrix' "
-                f"of shape {adjacency_matrix_shape} "
-                "therefore 'node_coords' should be a array "
-                f"with shape ({adjacency_matrix_shape[0]}, 3) "
-                f"while its shape was {node_coords_shape}"
-            )
-
-            raise ValueError(message)
-
-        if (
-            isinstance(node_color, (list, np.ndarray))
-            and len(node_color) != 1
-            and len(node_color) != node_coords_shape[0]
-        ):
-            raise ValueError(
-                "Mismatch between the number of nodes "
-                f"({node_coords_shape[0]}) "
-                f"and the number of node colors ({len(node_color)})."
-            )
-
-        if node_coords_shape[0] != adjacency_matrix_shape[0]:
-            raise ValueError(
-                "Shape mismatch between 'adjacency_matrix' "
-                "and 'node_coords'"
-                f"'adjacency_matrix' shape is {adjacency_matrix_shape}, "
-                f"'node_coords' shape is {node_coords_shape}"
-            )
+        # Validate inputs
+        self.input_checks(
```

Comment:
```suggestion
        self._check_inputs_add_graph(
```

comment should now be unnecessary given the name of the method

Reply:
Right!